packages/discord.js/src/util/Sweepers.js

```diff
@@ -425,6 +425,9 @@ class Sweepers {
     let items = 0;
 
     for (const guild of this.client.guilds.cache.values()) {
+      // We may be unable to sweep the cache if the guild is unavailable and was never patched
```

Comment:
I'd rather skip sweeping entirely if guild.available is false instead of checking if a key is present, but others might disagree (so lets discuss it!)

Reply:
I'm with Vlad here.

Reply:
Fixed!